api/registrations/serializers.py

```diff
@@ -12,13 +12,12 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
                                  help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    embargo_end_date = HideIfRetraction(ser.SerializerMethodField(help_text='When will embargo be lifted?'))
```

Comment:
One of the serializer fields in question (`pending_embargo`) comes from the property `Node.is_pending_embargo`, which does a check on the nodes/parents associated `Embargo` object. 

The field `pending_registration_approval` should perhaps come from `Node.is_pending_registration`, which does a check on the nodes/parents associated `RegistrationApproval` object, but it actually comes from `Node.sanction.is_pending_approval`, based on the `Sanction` associated with a given `Node`.